### DISCUSSION TITLE
parte 2 pregunta 5

### DISCUSSION THREAD

**Discussion by santiago3c** [IIC2133-PUC/2024-2#17] [Tarea 0] [answered]
Hola, una pregunta, en el rango [end-start], ¿es posible que end y start sean iguales, por ejemplo end = 9 y start = 9?

**Comment by JavierLarre** [comment, marked as answer]
Hola! si start es 9 entonces end es **por lo menos 10** , es imposible que start sea igual que end. Suerte en la tarea!

**Reply by sofiagutierrezleiva** [reply]
Hola! Entonces siempre que se regala es más de 1 puffle?

**Reply by JavierLarre** [reply]
Correcto. Si start y end son 9 y 10, entonces los puffles 9 y 10 son regalados

**Comment by santiago3c** [comment]
gracias
